docs/advanced/addons/managed-dhcp.md

```diff
@@ -0,0 +1,252 @@
+---
+sidebar_position: 6
+sidebar_label: Managed DHCP
+title: "Managed DHCP"
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.3/advanced/managed-dhcp"/>
+</head>
+
+_Available as of v1.3.0_
+
+Beginning with v1.3.0, you can configure IP pool information and serve IP addresses to VMs running on Harvester clusters using the embedded Managed DHCP feature. This feature, which is an alternative to the standalone DHCP server, leverages the vm-dhcp-controller add-on to simplify cluster deployment.
```

Comment:
```suggestion
Beginning with v1.3.0, you can configure IP pool information and serve IP addresses to VMs running on Harvester clusters using the embedded Managed DHCP feature. This feature, which is an alternative to the standalone DHCP server, leverages the vm-dhcp-controller add-on to simplify guest cluster deployment.
```

Reply:
nit. just to let the user know it's useful for VMs.

Reply:
Updated, thanks for the suggestion!